endoreg_db/views/Frames_NICE_and_PARIS_classifications_views.py

```
@@ -3,7 +3,9 @@
 from rest_framework.response import Response
 from rest_framework import status
 from ..models import VideoFile
-# from ..serializers.Frames_NICE_and_PARIS_classifications import ForNiceClassificationSerializer
+#from ..serializers.Frames_NICE_and_PARIS_classifications import ForNiceClassificationSerializer
+from ..serializers._old.Frames_NICE_and_PARIS_classifications import ForNiceClassificationSerializer
```

Ruff (0.8.2)

`..serializers._old.Frames_NICE_and_PARIS_classifications.ForNiceClassificationSerializer` imported but unused

Remove unused import: `..serializers._old.Frames_NICE_and_PARIS_classifications.ForNiceClassificationSerializer`

(F401)